From: Gianfranco Costamagna <locutusofborg@debian.org>
Date: Tue, 7 Jan 2020 09:08:34 +0100
Subject: This test seems to require an active AMQP instance,

Bug-Debian: https://bugs.debian.org/911693
Last-Update: 2018-12-17

something not possible during build process. Disable it for now.
---
 tests/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
index cf042b0..0f622cd 100644
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -34,7 +34,7 @@ add_executable(test_basic
 target_link_libraries(test_basic rabbitmq-static)
 
 if (NOT APPLE)
-  add_test(basic test_basic)
+  #add_test(basic test_basic)
 endif()
 
 add_executable(test_sasl_mechanism test_sasl_mechanism.c)
